resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -22,17 +22,20 @@ trait DbReader[F[_]] {
 object DbReader {
   implicit def apply[F[_]](implicit ev: DbReader[F]): DbReader[F] = ev
 
-  val deletedDisksQuery =
+  val shouldBedeletedDisksQuery =
     sql"""
            SELECT pd1.id, pd1.cloudContext, pd1.cloudProvider, pd1.name, pd1.zone, pd1.formattedBy
            FROM PERSISTENT_DISK AS pd1
-           WHERE pd1.status="Deleted" AND
-           pd1.destroyedDate > now() - INTERVAL 30 DAY AND
+           WHERE
+           (
+             (pd1.status="Deleted" AND pd1.destroyedDate > now() - INTERVAL 30 DAY) OR 
+             (pd1.status="Deleting" AND pd1.`dateAccessed` < now() - INTERVAL 30 MINUTE)
```

Comment:
if a disk is stuck in "Deleting" status for more than 30 min, we consider this an anomaly and resource validator will intervene and delete it